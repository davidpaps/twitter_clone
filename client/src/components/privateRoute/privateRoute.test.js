import React from "react";
import { shallow } from "enzyme";
import PrivateRoute from "./privateRoute";

describe("Private Route", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<PrivateRoute />)));

  it("should render a <Route />", () => {
    expect(wrapper.find("Route").length).toEqual(1);
  });

  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
