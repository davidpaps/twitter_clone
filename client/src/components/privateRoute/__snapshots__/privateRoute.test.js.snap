// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Private Route should render correctly 1`] = `ShallowWrapper {}`;
